frontend/src/components/pages/student/AssessmentSummaryPage.tsx

```diff
@@ -57,11 +63,24 @@ const AssessmentSummaryPage = (): React.ReactElement => {
               <Button onClick={handleBackToHome} variant="secondary">
                 Back to Home
               </Button>
-              <Button variant="primary">Start Test</Button>
+              <Button
+                onClick={() => setShowBeginTestModal(true)}
+                variant="primary"
+              >
+                Start Test
+              </Button>
             </HStack>
           </VStack>
         </HStack>
       )}
+      <Modal
+        body="Once you start, you may not exit the browser or it will end the test automatically."
+        header="Begin the test?"
+        isOpen={showBeginTestModal}
+        onCancel={() => setShowBeginTestModal(false)}
```

Comment:
this my fault but i just realized we don't need both `onCancel` and `onClose`. can we do a quick refactor to remove `onCancel` entirely and set its value to be `onClose`? in `RemoveUserModal`, we don't need the `onCloseParent` [thing](https://github.com/uwblueprint/jump-math/blob/staging/frontend/src/components/user-management/RemoveUserModal.tsx#L69)

Reply:
good catch! yes